src/matrix/inversewishart.jl

```diff
@@ -15,12 +15,15 @@ function InverseWishart{T<:Real}(df::T, Ψ::AbstractPDMat{T})
     p = dim(Ψ)
     df > p - 1 || error("df should be greater than dim - 1.")
     c0 = _invwishart_c0(df, Ψ)
-    R = promote_type(T, typeof(c0))
-    _, prom_Ψ = promote_eltype(one(R), Ψ)
+    R = Base.promote_eltype(T, c0)
+    prom_Ψ = convert(AbstractArray{R}, Ψ)
```

Comment:
This is the Base idiom for converting the element type of an array regardless of the array type. In this case, it will be some kind of `PDMat`.